Tests/UrlCreatorTests.cs

```diff
@@ -0,0 +1,22 @@
+using App.Services.ApiRequest;
+using NUnit.Framework;
+
+namespace Tests
+{
+    public class UrlCreatorTests
+    {
+        [Test]
+        public void GetBaseUri_RetunsBaseUriString()
+        {
+            // Arrange
+            IUrlCreator urlCreator = new UrlCreator();
+            string expected = "https://content.guardianapis.com/search?show-blocks=all&api-key=7a75b617-4388-40dc-80f3-b15f4e0e42f6";
```

Comment:
Don't store the API keys.  Use https://docs.microsoft.com/en-us/dotnet/architecture/microservices/secure-net-microservices-web-applications/developer-app-secrets-storage